### PR TITLE
Added day/night commands and syncing bed behavior

### PR DESCRIPTION
**Pull request by DerTyan**
Also blocked the day/night commands on the client.

### CODE REVIEW

File: NitroxPatcher/Patches/DayNightCycle_OnConsoleCommand_night_Patch.cs [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+    class DayNightCycle_OnConsoleCommand_night_Patch : NitroxPatch
+    {
+        public static bool Prefix()
+        {
+            return false;
+        }
+
+        public override void Patch(HarmonyInstance harmony)
+        {
+            PatchPrefix(harmony, typeof(DayNightCycle).GetMethod("OnConsoleCommand_night", BindingFlags.Instance | BindingFlags.NonPublic));
```
[the remark this answers is not in the document]
Comment (DerTyan) [reply NitroxPatcher/Patches/DayNightCycle_OnConsoleCommand_night_Patch.cs]: Updated!

File: NitroxPatcher/Patches/DayNightCycle_OnConsoleCommand_day_Patch.cs [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+namespace NitroxPatcher.Patches
+{
+    class DayNightCycle_OnConsoleCommand_day_Patch : NitroxPatch
+    {
+        public static readonly Type TARGET_CLASS = typeof(DayNightCycle);
+        public static readonly MethodInfo TARGET_METHOD = TARGET_CLASS.GetMethod("OnConsoleCommand_day", BindingFlags.Instance | BindingFlags.NonPublic);
+
+        public static bool Prefix()
+        {
+            return false;
```
Comment (Sunrunner37) [inline NitroxPatcher/Patches/DayNightCycle_OnConsoleCommand_day_Patch.cs]: We can block these but should we send the corresponding packets?  (if the users doesn't have permission the server will just let them know.

**Comment by Sunrunner37** [comment]
Merging in for now, can you address the item above in another PR?

**Comment by DerTyan** [comment]
Of course!

**Pull request merged**